auth.go

```diff
@@ -61,9 +62,12 @@ type Auth struct {
 	banned         *set.Set
 	allowlist      *set.Set
 	ops            *set.Set
+	keynamesByFingerprint map[string]string
```

Comment:
Would it be better to make this `fingerprintComments`, and store the comment section of the fingerprint altogether here? This way we can extract both the name and the symbol later.

Reply:
My intent was to just parse the keyname (first word) from the comment, and drop the rest. I'd rather leave future features to future developers.